extensions/Needinfo/template/en/default/hook/request/email-after_summary.txt.tmpl

```diff
@@ -8,6 +8,27 @@
 
 [% RETURN UNLESS flag && flag.type.name == 'needinfo' && flag.status == '?' %]
 ---
---- This request has set a needinfo flag on the [% terms.bug %].
---- You can clear it by logging in and replying in a comment.
+--- Hello from [% terms.Bugzilla %], the [% terms.bug %] tracker for Mozilla and Firefox:
+---
+--- Someone has asked for you for information about this [% terms.bug %].
+---
+--- Please go to [%+ urlbase %]show_bug.cgi?id=[% bug.bug_id %] to see the
```

Comment:
nit: pretty sure you don't need the `+` in `[%+ urlbase %]`; I don't think that template toolkit will trim the leading space there.

Reply:
You caught my copy-pasta from the main template in request :blush: I'll remove that and read the fine docs. :grin: